src/main/java/com/internet/shop/controllers/user/LogoutController.java

```diff
@@ -0,0 +1,16 @@
+package com.internet.shop.controllers.user;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class LogoutController extends HttpServlet {
```

Comment:
this should be in the next PR

Reply:
Did it fo the debagging? Can I leave it?

Reply:
Deleted